### ISSUE TITLE
Browser bundle error wtih Webpack

### ISSUE THREAD

**Issue by pkulcsarsz** [googleapis/google-api-nodejs-client#2503]
Based on the [documentation](https://github.com/googleapis/google-api-nodejs-client/tree/master/src/apis/sheets) on GoogleApi Sheets the next steps should create a bundle sheets.min.js. The process is throwing errors.

#### Environment details

  - OS: macOS 10.15.7
  - Node.js version: v12.19.1 / v14.15.1
  - npm version: 6.14.8
  - `googleapis` version: 
  - "googleapis-common": "^5.0.0"
  -  "webpack": "^5.0.0",
  -  "webpack-cli": "^4.0.0"

#### Steps to reproduce (based on documentation)

- cd src/apis/sheets
- npm install
- npm run webpack

#### Error:
```

 npm run webpack

> @googleapis/sheets@0.1.0 webpack /Users/myuser/google-api-nodejs-client/src/apis/sheets
> webpack

[webpack-cli] Invalid configuration object. Webpack has been initialized using a configuration object that does not match the API schema.
 - configuration.node should be one of these:
   false | object { __dirname?, __filename?, global? }
   -> Include polyfills or mocks for various node stuff.
   Details:
    * configuration.node has an unknown property 'child_process'. These properties are valid:
      object { __dirname?, __filename?, global? }
      -> Options object for node compatibility features.
    * configuration.node has an unknown property 'fs'. These properties are valid:
      object { __dirname?, __filename?, global? }
      -> Options object for node compatibility features.
    * configuration.node has an unknown property 'crypto'. These properties are valid:
      object { __dirname?, __filename?, global? }
      -> Options object for node compatibility features.
    * configuration.node has an unknown property 'http2'. These properties are valid:
      object { __dirname?, __filename?, global? }
      -> Options object for node compatibility features.
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! @googleapis/sheets@0.1.0 webpack: `webpack`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the @googleapis/sheets@0.1.0 webpack script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
```

**Comment by sofisl** [comment]
Hi @pkulcsarsz, we haven't yet upgraded this library to webpack 5, but there is a PR (#2480) out with the change. Can you try bundling the library with an earlier version of webpack?

**Comment by pkulcsarsz** [comment]
Hi @sofisl. 
I have successfully managed to build the library with webpack 4 for node usage. Even though it is not possible to import the exported values (sheets, auth) from the generated file. Is there any way to pack the sheets library only so it can be used in a serverless world, where the size of the imported modules is a key factor in the cold start.

Thank you

**Comment by SurferJeffAtGoogle** [comment]
This issue will fall OoSLO soon.

**Comment by sofisl** [comment]
This should be fixed in #2524
